### ISSUE TITLE
Change global_params to context , state, global_state 

### ISSUE THREAD

**Issue by balthazarneveu** [balthazarneveu/interactive_pipe#47]
Backward compatibility: allow several names if possible

**Comment by balthazarneveu** [comment]
Fixed in v7.5 

Supported keywords (equivalently)
 ```python
"global_params", "global_parameters", "global_state", "global_context", "context", "state"
```

**Comment by balthazarneveu** [comment]
![image](https://github.com/user-attachments/assets/9d901a1e-6847-4906-9b6f-43f1071cff43)

after v0.7.5

should see 42
![image](https://github.com/user-attachments/assets/6a815610-63f0-4922-ba5a-23e5223c47cd)

**Comment by balthazarneveu** [comment]
Fixed in v7.5 

Supported keywords (equivalently)
 ```python
"global_params", "global_parameters", "global_state", "global_context", "context", "state"
```
